src/main/scala/in/ashwanthkumar/suuchi/partitioner/PartitionNode.scala

```diff
@@ -0,0 +1,6 @@
+package in.ashwanthkumar.suuchi.partitioner
+
+case class Node(id: Int, host: String)
```

Comment:
Thinking more - should we reuse the existing `MemberAddress` for `Node` ?  or re-use `Member` and `MemberAddress` inside `Node` ?